libvast/vast/system/archive.hpp

```diff
@@ -30,8 +30,8 @@ struct archive_state {
   void send_report();
   void next_session();
   archive_actor::pointer self;
-  std::unique_ptr<vast::store> store;
-  std::unique_ptr<vast::store::lookup> session;
+  std::unique_ptr<vast::segment_store> store;
+  std::unique_ptr<vast::segment_store::lookup> session;
```

Comment:
Pretty sure these do not need to be wrapped in `unique_ptr` anymore. Up to you if you want to spend time on that, though.

Reply:
I'm aware, but I'm avoiding changes that aren't strictly necessary in order to keep the diff small at the moment.

Reply:
Makes sense, I kinda figured that was the reasoning.